scripts/stac/imagery/create_stac.py

```diff
@@ -130,6 +130,9 @@ def create_item(
         gdalinfo_result = gdal_info(asset_path)
 
     if derived_from is not None:
+        # Remove existing derived_from links in case of resupply
+        item.stac["links"] = [link for link in item.stac["links"] if link["rel"] != "derived_from"]
```

Comment:
might be better to move this outside of `if derived_from is not None:`.

Reply:
As discussed, from business pov there should not be an issue as I don't think an existing Item that is derived from would become an Item from its own, but technically that can happen.